### DISCUSSION TITLE
agentfabric项目，如何调用本地文生图API？

### DISCUSSION THREAD

**Discussion by cgwyx** [modelscope/ms-agent#355] [Q&A]
agentfabric项目，想调用本地文生图API,根据文档：Agent实操（二）：低代码调用API创建更加酷炫的Agent（https://www.modelscope.cn/headlines/article/267）,
怎么设置也无法调用本地的文生图API,是框架不支持吗，还是设置问题？各位有做过的吗，求指导

**Comment by zzhangpurdue** [comment]
你可以本地部署一个api，并新开发一个tool去调用他，可以参考这个[文档](https://mp.weixin.qq.com/s?__biz=MzkxNTM5NTg2OA==&mid=2247490552&idx=1&sn=57f57a68a6aade970b53140451d5669b&chksm=c15e9f8bf629169d54035b56f63231ffd0eda4db0324043bd0b70a4ba41833bcc4170eaa27f3&mpshare=1&scene=1&srcid=0314FXemoX6s1UF1BnLuOAg8&sharer_shareinfo=97a071b44cb4eaba6eb3d3db6c8e20dc&sharer_shareinfo_first=97a071b44cb4eaba6eb3d3db6c8e20dc&version=4.1.20.90880&platform=mac#rd) 的最后如何生成一个tool的部分
